server/app/lib/analysis_library/schema/osa.json

```diff
@@ -0,0 +1,1106 @@
+					"type": "string",
+                    "default": "true",
+					"enum": [
+						"true", "false"
+					]
+				},
+				"sample_method": {
+					"description": "The type of sampling. If 'all_variables' then the sampling will occur across all the variables. If individual, then each variable is sampled individually.",
+					"type": "string",
+                    "default": "individual_variables",
+					"enum": [
+						"all_variables", "individual_variables"
+					]
+				},
+				"seed": {
+					"description": "Seed value for random number generators. If null then algorithm will choose random seed.",
+					"type": [
+						"number", "null"
+					]
+				},
+                "solution_tolerance": {
+                    "description": "RGenoud: Numbers within solutionTolerance are considered equal",
+					"type": "number",
+                    "default": 0.01,
+                    "minimum": 0
+				},
+                "topology": {
+                    "description": "PSO: Options: gbest, lbest, vonneumann, random"
+					"type": "string",
+                    "default": "random",
+                    "enum": [
+						"gbest", "lbest", "vonneumann", "random"
+					]
+				},
+                "tournament_size": {
+                    "description": "NSGA2, SPEA2: Tournament Size",
+					"type": "number",
+                    "default": 2
+				},
+                "toursize": {
+					"type": "number"
+				},
+                "type": {
+                    "description": "Sobol: method",
+					"type": "string",
+                    "default": "sobol",
+                    "enum": [
+						"sobol", "sobol2002", "sobol2007", "jansen", "mara", "martinez"
+					]
+				},
+                "type": {
+                    "description": "Morris: Options: oat (One at a Time) TODO: only one and conflicts with sobol",
+					"type": "string",
+                    "default": "oat",
+                    "enum": [
+						"oat"
+					]
+				},
+                "vini": {
+                    "description": "PSO: Options: zero, lhs2011, random2011, lhs2007, random2007, default"
+					"type": "string",
+                    "default": "lhs2011",
+                    "enum": [
+						"zero", "lhs2011", "random2011", "lhs2007", "random2007", "default"
+					]
+				},
+				"wait_generations": {
+                    "description": "RGenoud: If no improvement in waitGenerations of generations, then exit",
+					"type": "number",
+                    "default": 2,
+                    "minimum": 0
+				},
+				"xini": {
+                    "description": "PSO: Options: lhs, random"
+					"type": "string",
+                    "default": "lhs",
+                    "enum": [
+						"lhs", "random"
+					]
+				},
+                "xover_dist_idx": {
+                    "description": "NSGA2: Crossover Distribution Index (large values give higher probabilities of offspring close to parent)",
+					"type": "number",
+                    "default": 5
+				},
+				"xoverdistidx": {

```

Comment:
is this used?

Reply:
I think we leave these for now even if they aren't used.